data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -1,14 +1,26 @@
+from collections import namedtuple
+import os
 import pandas as pd
+import numpy as np
+import geopandas as gpd
 
+from data_pipeline.config import settings
 from data_pipeline.etl.base import ExtractTransformLoad
 from data_pipeline.etl.sources.census_acs.etl_utils import (
     retrieve_census_acs_data,
+    impute_by_geographic_neighbors,
 )
-from data_pipeline.utils import get_module_logger
+
+from data_pipeline.utils import get_module_logger, unzip_file_from_url
 from data_pipeline.score import field_names
 
 logger = get_module_logger(__name__)
 
+# because now there is a requirement for the us.json, this will port from
+# AWS when a local copy does not exist.
+# @Jorge -- is this too much of a hack?
+CENSUS_DATA_S3_URL = settings.AWS_JUSTICE40_DATASOURCES_URL + "/census.zip"
```

Comment:
Jorge -- this is not the most elegant. I'm wondering if you had a better solution?